python/src/ecole/core/dynamics.cpp

```diff
@@ -263,6 +264,14 @@ void bind_submodule(pybind11::module_ const& m) {
 					The source of randomness. Passed by the environment.
 		)")
 		.def(py::init<>());
+
+	dynamics_class<PrimalSearchDynamics>(m, "PrimalSearchDynamics")  //
+		.def(
```

Comment:
Missing bindings to the other methods, no? Also can you had some docstrings.

Reply:
Yes that's still missing, the PR is incomplete.